API_Run_of_Connectors/API_run_Connector_multiple_files.rb

```diff
@@ -1,85 +1,80 @@
+# frozen_string_literal: true
+
 # kenna run connectors
 require 'rest-client'
 require 'json'
 
-
-
 @token = ARGV[0]
 @folder = ARGV[1]
 @connector_id = ARGV[2]
 @file_extension = ARGV[3]
 
-
-@LOCATOR_DELIMITER = ":"
-@API_ENDPOINT_CONNECTOR = "https://api.kennasecurity.com/connectors"
-@headers = {'content-type' => 'application/json', 'X-Risk-Token' => @token, 'accept' => 'application/json'}
+@LOCATOR_DELIMITER = ':'
+@API_ENDPOINT_CONNECTOR = 'https://api.kennasecurity.com/connectors'
+@headers = { 'content-type' => 'application/json', 'X-Risk-Token' => @token, 'accept' => 'application/json' }
```

Comment:
the keys are strings, not symbols, so that change would be potentially breaking.

These strings are awkward to represent as symbols, too. the `-` and `/` mean it has to be `'content-type': `. They still have to be quoted.

Reply:
i believe we use the syntax `'key': value` elsewhere in the app, so i think it _should_ be ok but i haven't tried it in this case.

Reply:
Sure, the syntax is valid, it's just not much of an improvement. 

The real issue is that the keys are strings and that syntax gives you symbol keys (even if it's `'key': value`- that key is a symbol) and without knowing about the usage of the hash, you can't change from string keys to symbol keys without potential breaks.